packages/caip/src/constants.ts

```diff
@@ -25,6 +25,8 @@ export const foxatarAssetId: AssetId =
   'eip155:137/erc721:0x2e727c425a11ce6b8819b3004db332c12d2af2a2'
 export const foxyAssetId: AssetId = 'eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3'
 
+export const usdtAssetId: AssetId = 'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7'
+
```

Comment:
Since we already need this in a few places, and are going to need it even more so in the follow-up PRs (e.g https://github.com/shapeshift/web/pull/7665/files#diff-a3476567ba166de3cf5ef24f5767d4c2fee3c4ba2b07df98ced5d21f31d93c4bR15), might as well